src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -198,6 +198,22 @@ protected override async Task<bool> AcceptEvent(SessionId sessionId, JObject par
                                     await SendResume(sessionId, token);
                                     return true;
                                 }
+                            case "mono_wasm_fire_debugger_agent_message_with_data":
+                            case "_mono_wasm_fire_debugger_agent_message_with_data":
```

Comment:
How does `_mono_wasm_fire_debugger_agent_message_with_data` originate and can't we force to have only e.g. `mono_wasm_fire_debugger_agent_message_with_data`?

Reply:
Never mind, I just noticed all the cases are in this convention.